### PR TITLE
Fixed ccm 0.6.0 CCM deps and CCM script logic

### LINKED ISSUE #39: Unattended Mode option
It would be great to have a method for triggering the entire QuickStart process, without having to run each script one-by-one. This would make testing easier as well.

### LINKED ISSUE #48: CCM 0.6.0 Package installs failing
As the Chocolatey Central Management packages have been updated to v `0.6.0`, we now have different dependencies required at different steps in the process as well.

Firstly the following packages now require a minimum of v `3.1.16`:
- `aspnetcore-runtimepackagestore`
- `dotnetcore-windowshosting`

Also, as a result of moving to aspnet-core as a requirement, the above dependencies are now required earlier on in the scripting. Before, only CCM Web required the above. Now, CCM DB and CCM Service also depend on them.

All the above logic will need to be added to the `Start-C4bCcmSetup.ps1` script.

### PR DESCRIPTION
**Pull request by adilio**
Closes #48 .

Bugfix:
- With CCM updates to v 0.6.0, new dependencies defined:
  - `aspnetcore-runtimepackagestore 3.1.16`
  - `dotnetcore-windowshosting 3.1.16
- Update logic to move above package installs up before CCM DB and Service installs (as they now require the above dependencies)
- Updates to README doc
- Update script name casing to match PowerShell conventions

### CODE REVIEW

**Comment by steviecoaster** [comment]
Probably shouldn't do all this in one PR. CCM dependencies uplift in 1, unattended in another. Keeps the git history clean.

**Comment by adilio** [comment]
Understood @steviecoaster.

I've removed the `Unattended` stuff from this PR by reverting those changes in my last commit.

**Review by ryanrichter94 — APPROVED** [review]

File: Start-C4bSetup1.ps1 [hunk 1]
```diff
@@ -18,13 +18,44 @@ param(
...
+    [string]
+    $Thumbprint
 )
 
 $DefaultEap = $ErrorActionPreference
 $ErrorActionPreference = 'Stop'
 Start-Transcript -Path "$env:SystemDrive\choco-setup\logs\Start-C4bSetup-$(Get-Date -Format 'yyyyMMdd-HHmmss').txt"
 
+# If variables for Unattend are not defined, prompt for them.
+if ($Unattend) {
```
Comment (steviecoaster) [inline Start-C4bSetup1.ps1]: Eh? Thought we yanked this.

**Review by steviecoaster — APPROVED** [review]

File: Start-C4bSetup1.ps1 (hunk 1 above)
Comment (steviecoaster) [reply Start-C4bSetup1.ps1]: This is fine. It was removed in the last commit

**Pull request merged**